README.md

```diff
@@ -5,13 +5,14 @@
 - Modalità: **CLASSIC**
 
 ### Montepremi
-* Quota di partecipazione: **50€**
-* Campionato
-	* 1° posto: **200€** (50% del totale)
-	* 2° posto: **100€** (25% del totale)
-	* 3° posto: **50€** (12.5% del totale)
-* Coppa:
-	* Vincitore: **50€** (12.5% del totale)
+* Quota di partecipazione: **60€**
+* Campionato andata e ritorno
+	* 1° posto: **130€** (27% del totale)
+	* 2° posto: **70€** (14% del totale)
+* Coppe:
+	* Vincitore Champion's: **40€** (8% del totale)
+	* Vincitore Europa League: **20€** (4% del totale)
+	* Vincitore Supercoppa: **20€** (4% del totale)
```

Comment:
le coppe mi sembrano over-premiate trattandosi di tre partite l'una. io farei 25, 15, 5. avanzano 35 che ridistribuirei al terzo dei campionati: 120 al primo, 65 al secondo, 30 al terzo oppure 125, 65, 20

Reply:
sono d'accordo ma 25-15-5 sono pochi secondo me.
propongo quindi 25-20-15

Reply:
onestamente 15 per la supercoppa mi sembra veramente troppo, e 25-20 non fa abbastanza differenza tra CL e EL. 30, 20, 10?

Reply:
@jackthebeast si potrebbe fare. considera che questo cambio lo vorrei fare per far sì che il 7° e l'8° mettano la formazione. quindi potenzialmente un 8° può vincere europa e supercoppa e in tutto sono 30€. certo, 35€ non è molto di più ma è più parente di 40 🙂 
se il dibattere è solo sulla vincita per me va bene possiamo cambiare in 30-20-10

Reply:
sisi a me la formula piace. come ho detto aggiungerei anche una sorta di finale tra i due vincitori dei campionati. ma non so se abbiamo molto senso visto che se sono diversi è probabile che quello della clausura sia "più forte" in quel momento.

le vincite le possiamo discutere a parte cmq.